resources/config/io.neonbee.internal.verticle.ServerVerticle.example.yaml

```diff
@@ -46,6 +46,10 @@ config:
         authenticationChain: ~
         # namespace and service name URI mapping (strict, or loose based on CDS)
         uriConversion: strict
+        # a block / allow list of verticles to expose via this endpoint (defaults to empty / all entities exposed)
```

Comment:
It should be mentioned that the value of block and allow attribute should be an array with regexp entries.